canisters/management.ts

```diff
@@ -150,6 +150,36 @@ export type HttpResponse = {
     body: blob;
 };
 
+export type KeyID = {
```

Comment:
Teeny-tiny thing... I think we want `KeyId`, not `KeyID`. Just like `Ecdsa`, I think we only capitalize the first letter of an abbreviation/acronym.